src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -52,11 +52,26 @@ file that was distributed with this source code.
             </div>
             {% if results_count > 0 %}
                 <div class="box-body no-padding">
-                    <ul class="nav nav-stacked sonata-search-result-list">
+                    <ul class="nav nav-stacked sonata-search-result-list products-list">
                         {% for result in current_page_results %}
                             {% set link = admin.getSearchResultLink(result) %}
                             {% if link %}
-                                <li><a href="{{ link }}">{{ admin.toString(result) }}</a></li>
+                                <li class="item no-padding">
+                                    <a class="main pull-left" href="{{ link }}">{{ admin.toString(result) }}</a>
+                                    <span class="matches pull-right">
+                                        {% for name, filter in filters %}
+                                            {% set match = name|split('.')|reduce((carry, v) => carry ? attribute(carry, v) : null, result) %}
+
+                                            {% if match and (((filter.options.case_sensitive or filter.options.case_sensitive is null) and filter.value.value in match)
```

Comment:
Given the behavior from MySQL about case sensitivity (see sonata-project/SonataDoctrineORMAdminBundle#1395), I don't know if we should use a case sensitive matching here, it that can behave different than the database comparison.